fieldservice_account/security/ir.model.access.csv

```diff
@@ -1,2 +1,6 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_account_invoice_line_fsm_user,account.invoice.line.fsm.user,model_account_invoice_line,fieldservice.group_fsm_user,1,1,1,1
+access_account_invoice_line_portal,access.account.invoice.line.portal,account.model_account_invoice_line,base.group_portal,1,0,0,0
+access_account_account_portal,access.account.account.portal,account.model_account_account,base.group_portal,1,0,0,0
+access_account_move_portal,access.account.move.portal,account.model_account_move,base.group_portal,1,0,0,0
+access_account_move_line_portal,access.account.move.line.portal,account.model_account_move_line,base.group_portal,1,0,0,0
```

Comment:
I'm curious why we need to grant these permissions for these models from accounting?

Reply:
@bodedra I have security concerns with this - it makes the Accounting moves an open book for any Portal user. I would vote against adding this.

Reply:
@dreispt I agree with you. @max3903 @murtuzasaleh would you please provide your input?

Reply:
https://github.com/OCA/field-service/pull/615#discussion_r476673789 - See similar issue and responses here.  Considering these changes are for the mobile app, shouldn't they go in the repo for the mobile app? (which I'm assuming is not going to be OCA)